src/pages/workspace/workflows/WorkspaceWorkflowsPage.tsx

```diff
@@ -86,9 +83,12 @@ function WorkspaceWorkflowsPage({policy, betas, route, reimbursementAccount, ses
     }, []);
 
     const optionItems: ToggleSettingOptionRowProps[] = useMemo(() => {
-        const {accountNumber, state, bankName} = reimbursementAccount?.achData ?? {};
-        const hasVBA = state === BankAccount.STATE.OPEN;
-        const bankDisplayName = bankName ? `${bankName} ${accountNumber ? `${accountNumber.slice(-5)}` : ''}` : '';
+        const {accountNumber, addressName} = policy?.achAccount ?? {};
+        const hasVBA = !!policy?.achAccount;
+        let bankDisplayName = addressName;
+        if (accountNumber && bankDisplayName !== accountNumber) {
+            bankDisplayName += accountNumber.slice(-5);
```

Comment:
I think we need to add a space in between, no?
![Screenshot 2024-03-26 at 9 56 20 AM](https://github.com/Expensify/App/assets/22219519/09eb96e2-0293-4a98-8dc1-caae32fe739b)

Also, just noting that we used to show the bank name instead of addressName, e.g.
![Screenshot 2024-03-26 at 9 42 51 AM](https://github.com/Expensify/App/assets/22219519/912eb7f5-6172-42cd-9abb-7abdeb082ad9)

I don't think that's a blocker but seems to be different than what we had in the design doc.

Reply:
Good catch. I've committed a fix, but we'll need https://github.com/Expensify/Auth/pull/10335/files for it to really work. Not a blocker though